News/App.xaml.cs

```diff
@@ -100,11 +115,54 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         /// </summary>
         /// <param name="sender">Источник запроса приостановки.</param>
         /// <param name="e">Сведения о запросе приостановки.</param>
-        private void OnSuspending(object sender, SuspendingEventArgs e)
+        private  void OnSuspending(object sender, SuspendingEventArgs e)
         {
+            WriteUserCheckedListElements();//json user data to file
+
             var deferral = e.SuspendingOperation.GetDeferral();
             //TODO: Сохранить состояние приложения и остановить все фоновые операции
             deferral.Complete();
         }
+
+
+        async void WriteUserCheckedListElements()
```

Comment:
Always use explicit access modifiers

Reply:
Also it is a good practice to add "Async" to the end of any async function's name (like "WriteUserCheckedListElementsAsync")